index.md

```diff
@@ -1,8 +1,15 @@
 # ПЕРЕД УХОДОМ НЕ ЗАБЫВАЕМ
 
+## ПРОВЕРИТЬ
+
+1. НАЛИЧИЕ ЛЮДЕЙ В ПОМЕЩЕНИЯХ
```

Comment:
We probably want to check absence, not presence

Reply:
Totally agree, @foxyfiction you really want to lock someone in the office?

Reply:
@maxmalov if you check the last commit you notice that I have fixed this item.